googlemock/include/gmock/gmock-matchers.h

```diff
@@ -2591,9 +2591,10 @@ class ContainerEqMatcher {
           UniversalPrint(*it, os);
         }
       }
+      return !printed_header2 && !printed_header;
```

Comment:
I don't understand. Why are these printed_header bools determining the truth value of the matcher?